### ISSUE TITLE
G0 - PDF attachments show image with no Alt text- Accessibility issue

### ISSUE THREAD

**Issue by anncesar** [brharp/hjckrrh#121]
pdf attachments show image with no Alt text
see here, bottom of page.
https://aoda-web.uoguelph.ca/ccs/gryph-mail

siteimprove doesn't like it.

**Comment by jpinsent** [comment]
This issue has been resolved. Alt text now matches title text.
